ui/src/components/RecordButton.js

```diff
@@ -53,7 +53,8 @@ export class RecordButton extends LitElement {
      * audio file.
      */
     async recordHandler() {
```

Comment:
nit: this is a huge function & I think we need to refactor it--might be best to do this in a later pr to avoid making this one too big